librad/src/paths.rs

```diff
@@ -81,22 +72,33 @@ impl Paths {
             git_includes_dir,
         } = self;
 
-        [
-            ("keys_dir", keys_dir.as_path()),
-            ("git_dir", git_dir.as_path()),
-            ("git_includes_dir", git_includes_dir.as_path()),
+        vec![
+            keys_dir.as_path(),
+            git_dir.as_path(),
+            git_includes_dir.as_path(),
         ]
-        .iter()
-        .cloned()
-        .collect()
+        .into_iter()
     }
 
     fn init(self) -> Result<Self, io::Error> {
-        self.all_dirs().values().try_for_each(fs::create_dir_all)?;
+        self.all_dirs().try_for_each(fs::create_dir_all)?;
         Ok(self)
     }
 }
 
+/// Returns [`ProjectDirs`] for this specific project (`radicle`).
+///
+/// Returns [`io::Error`] if the project directories could not be determined,
+/// most likely du to the `$HOME` environment variable missing
+pub(crate) fn project_dirs() -> Result<ProjectDirs, io::Error> {
+    ProjectDirs::from("xyz", "radicle", "radicle").ok_or_else(|| {
```

Comment:
Do we want to take the opportunity to change this to `xyz`, `radicle`, `radicle-link`? I'd think that migration logic would be greatly simplified, too.

Reply:
Good idea, since this is a breaking change anyway.

Reply:
Addressed in https://github.com/radicle-dev/radicle-link/pull/496/commits/79a81a3177810bf278b8ba73e644bd822aed573f